.github/scripts/generate_release_notes.py

```diff
@@ -14,10 +14,10 @@
 
 from github import Github
 
-milestoneProjectRegex = "^(.*) Milestone$"
+milestoneProjectRegex = "^(.*) Milestone .*$"
```

Comment:
Please, make the space after "Milestone" to be optional too, so the old style also works: "1.0.0 Milestone"

"^(.*) Milestone( [0-9])?$"